L1Trigger/L1THGCal/src/HGCalTriggerCellCalibration.cc

```diff
@@ -31,8 +38,17 @@ void HGCalTriggerCellCalibration::calibrateInMipT(l1t::HGCalTriggerCell& trgCell
   // fCperMIP and thickCorr here
   /* convert the charge amplitude in MIP: */
   double trgCellMipP = amplitude;
-  if (isSilicon && fCperMIP_ > 0) {
-    trgCellMipP /= fCperMIP_;
+
+  if (DetId(trgCell.detId()).det() == DetId::HGCalTrigger and
+      HGCalTriggerDetId(trgCell.detId()).subdet() == HGCalTriggerSubdetector::HFNoseTrigger) {
```

Comment:
If this kind of check is done in several places, it might be useful to add a function in the trigger tools class

Reply:
added the function in the triggertool